src/dry-run.ts

```diff
@@ -93,4 +93,5 @@ require('yargs')(hideBin(process.argv))
   })
   .demandCommand(1)
   .help()
+  .strict()
```

Comment:
Adding this option makes the dry run script show an error if you forget to give the "check" argument